docs/core-concepts/core-sdk/cli/dotnet-publish.md

```diff
@@ -0,0 +1,81 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--build-base-path] [--output]  
+    [--version-suffix] [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` compiles the application, reads through its dependencies specified in `project.json` and publishes the resulting set of files to a directory. 
+
+Depending on the type of portable app, the directory contains the following:
+
+1. **Portable application** - application's intermediate language (IL) code and all of application's managed dependencies.
+    * **Portable application with native dependencies** - as above with a sub-directory for each native dependencies' 
```

Comment:
for each native dependencies' supported platform. -> for the supported platform of each native dependency.